### ISSUE TITLE
Bug: App crashes when opening two folders at the same time

### ISSUE THREAD

**Issue by verdaderoken** [files-community/Files#10368]
### Description

Occurs whenever I try to open two folders at almost the same time using a script (AHK) or Run.
The app would sometimes immediately crash after the notification.

I'm planning to create a script where it opens specific folders when a shortcut key is triggered but I'm encountering these two errors:
`0x800706BA` & `0x800706BE`

The lowest delay I could set before executing the 2nd folder was 1.5 seconds.

### Steps To Reproduce

1. Set Files as default file manager
2. Open the Run dialog (Win+R)
3. Type in `C:\` and press enter
4. Quickly open the run dialog box and press enter again
5. See that the second path isn't opened in Files

### Steps to fix

- Fix the crash and open the second path as expected

### Files Version

2.3.56.0

### Windows Version

10.22623.875

### Relevant Assets/Logs

1. **The RPC server is unavailable**
```cs
2022-11-01 07:38:04.8938|ERROR|AppUnhandledException|A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (The RPC server is unavailable. (0x800706BA))
	System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (The RPC server is unavailable. (0x800706BA))
 ---> System.Runtime.InteropServices.COMException (0x800706BA): The RPC server is unavailable. (0x800706BA)
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|20_0(Int32 hr)
   at ABI.Microsoft.Windows.AppLifecycle.IAppActivationArgumentsMethods.get_Data(IObjectReference _obj)
   at Files.App.MainWindow.InitializeApplication(AppActivationArguments activatedEventArgs)
   at CommunityToolkit.WinUI.DispatcherQueueExtensions.<>c__DisplayClass2_0.<<EnqueueAsync>b__1>d.MoveNext()
   --- End of inner exception stack trace ---
```

**Comment by yaichenbaum** [comment]
Is this still an issue in v2.3.57 (should be available in around 15 minutes)?

**Comment by verdaderoken** [comment]
> Is this still an issue in v2.3.57 (should be available in around 15 minutes)?

Yes. I also tried this in cmd (Files is set as default):
```bat
start "" "C:\Windows"
start "" "C:\Users"
pause
```

Only either the **Users** folder or the **Windows** folder would open but not both.

**Comment by yaichenbaum** [comment]
I can reproduce this issue

**Comment by gave92** [comment]
Coudl you check if this issue is fixed in Files v2.3.62? #10575 might have helped.

**Comment by verdaderoken** [comment]
> Could you check if this issue is fixed in Files v2.3.62? #10575 might have helped.

using v2.3.63, the RPC thing is now gone and I can now run paths from a command line. However, sometimes I encounter these two errors:

```cs
2022-11-28 00:28:03.0926|ERROR|AppUnhandledException|The operation identifier is not valid. (0x800710DD)
	System.Runtime.InteropServices.COMException (0x800710DD): The operation identifier is not valid. (0x800710DD)
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|20_0(Int32 hr)
   at ABI.Microsoft.UI.Xaml.IWindowMethods.get_Content(IObjectReference _obj)
   at Files.App.Views.ModernShellPage.SetLoadingIndicatorForTabs(Boolean isLoading)
   at Files.App.Views.ModernShellPage.FilesystemViewModel_ItemLoadStatusChanged(Object sender, ItemLoadStatusChangedEventArgs e)
   at Files.App.ViewModels.ItemViewModel.RapidAddItemsToCollectionAsync(String path, String previousDir, Action postLoadCallback)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at Microsoft.UI.Dispatching.DispatcherQueueSynchronizationContext.<>c__DisplayClass2_0.<Post>b__0()
--- End of stack trace from previous location ---
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|20_0(Int32 hr)
   at ABI.Windows.ApplicationModel.Core.IUnhandledErrorMethods.Propagate(IObjectReference _obj)
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelperWinUI.<.ctor>b__0_3(Object sender, UnhandledErrorDetectedEventArgs eventArgs)
--- End of stack trace from previous location ---
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelperWinUI.<.ctor>b__0_3(Object sender, UnhandledErrorDetectedEventArgs eventArgs)
   at ABI.System.EventHandler`1.Do_Abi_Invoke[TAbi](Void* thisPtr, IntPtr sender, TAbi args)
   ```
  
  and this one
  
  ```cs
  2022-11-28 00:24:45.2935|ERROR|AppUnhandledException|A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (The process cannot access the file because it is being used by another process. (0x80070020))
	System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (The process cannot access the file because it is being used by another process. (0x80070020))
 ---> System.IO.FileLoadException: The process cannot access the file because it is being used by another process. (0x80070020)
   at Files.App.Helpers.UniversalLogWriter.InitializeAsync(String name)
   at Files.App.App.<>c.<<OnLaunched>b__84_0>d.MoveNext()
   --- End of inner exception stack trace ---
   ```
   
Both errors will not crash the app and only **some** of the folders are opened. 
My current workaround is to just add a delay before executing the subsequent folders.
